cmd/appliance/upgrade/complete.go

```diff
@@ -1,16 +1,14 @@
 package upgrade
```

Comment:
Diff here is big because most of the checks and preperations are moved to pkg/appliance/upgrade.go